FirebaseCore/Sources/Private/FIRAppInternal.h

```diff
@@ -89,6 +89,9 @@ extern NSString *const FIRAuthStateDidChangeInternalNotificationUIDKey;
 
 /**
  * A flag indicating if this is the default app (has the default app name).
+ *
+ * @note This property is referenced by selector name in `FirebaseAppInternal.swift`. If this
```

Comment:
Why `@note` here but `NOTE` above?

Reply:
The `NOTE` above are not documentation comments, just regular comments to call out to the reader. This one is a documentation comment so it'll show up in Xcode docs under the "Note" section since it's part of the header.

![Screen Shot 2022-01-18 at 12 59 04 PM](https://user-images.githubusercontent.com/1097316/149992741-23f31a7e-c5cc-4dbe-9752-59d271b207ad.png)